README.md

```diff
@@ -118,22 +116,19 @@ This library should only be used with the Okta management API. To call the [Auth
 
 ### Get a User
 ```
-config := okta.NewConfig()
-client := okta.NewClient(config, nil, nil)
+client := okta.NewClient(context, okta.WithOrgUrl("https://{yourOktaDomain}"), okta.WithToken("{apiToken}"))
```

Comment:
Probably we can assume that user already knows how to create `client`, so we don't need to duplicate this in all chapters

Reply:
This was decided on about a year ago to have copy/paste-able that included the client.